specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/examples/getQuotaRequestStatusById.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "resourceUrl": "management.azure.com/subscriptions/3f75fdf7-977e-44ad-990d-99f14f0f299f/providers/Microsoft.Compute/locations/eastus",
+    "id": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+    "api-version": "2021-03-15-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
```

Comment:
>2B5C8515-37D8-4B6A-879B-CD641A2CF605 [](http://example.com/codeflow?start=15&length=36)

this is not a valid fully qualified resource identifier, look [here](https://armwiki.azurewebsites.net/introduction/concepts/resourceids.html) in your case, I expect the id to be something like:

/providers/Microsoft.Quota/quotaLimitsRequests/{quotaRequestName}

Reply:
I will update the examples. It will be as - /subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{locationId}/providers/Microsoft.Quota/quotaLimitsRequests/{requestId}

The first part of the resource uri will depend on the RP, as explained earlier. The Batch RP will have batchAccountName after locations and CosmosDB will not have locations.

Reply:
I have updated the examples as suggested.